tekton/publish.yaml

```diff
@@ -144,6 +144,8 @@ spec:
 
       # Publish images and create release.yaml
       ko resolve --preserve-import-paths -t $(inputs.params.versionTag) -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > /workspace/output/bucket/latest/release.yaml
+      # Publish images and create release.notags.yaml
+      ko resolve --preserve-import-paths -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > /workspace/output/bucket/latest/release.notags.yaml
```

Comment:
I wonder if we could avoid re-building and re-publishing the images?

Reply:
It shouldn't rebuild them (nor re-publishing them) :angel: nothing should have changed from the first run to the other one so, the 2nd `ko resolve` should be pretty quick (at least that's what I think is happening :stuck_out_tongue_closed_eyes:)

Reply:
nice :) thanks